examples/linux64/hello-make/hello.c

```diff
@@ -0,0 +1,58 @@
+// Currently, it's not possible to specify executable entrypoint to ppci
+// linker - so far, the entrypoint is at the beginning of the code segment.
```

Comment:
I added the option to specify the entry point in the linker script by using:
```
ENTRY(startup)
```
Please refer to commit c078ba4310dec64600ddb78a35e79fead16cd43e for the change.

Thanks for this improvement hint! I'm pretty happy with this change!